creme/creme_core/templates/creme_core/forms/widgets/union.html

```diff
@@ -1,13 +1,11 @@
-<ul {% include "django/forms/widgets/attrs.html" %}>
+<div {% include "django/forms/widgets/attrs.html" %}>
 {% with attrs=widget.attrs %}
 {% for choice in widget.subwidget_choices %}
-  <li class="union-sub_widget">
-    <label>{# TODO: help text as tooltip ??? #}
-      <input class="union-widget-switch" type="radio" name="{{widget.name}}" value="{{choice.name}}"{% if widget.required %} required{% endif %}{% if choice.checked %} checked{% endif %}{% if widget.disabled %} disabled{% endif%}>
-      {{choice.label}}
-    </label>
-    <div class="union-sub_widget_container">{% include choice.widget.template_name with widget=choice.widget %}</div>
-  </li>
+  <div class="union-field">
+    <input class="union-field-toggle" type="radio" name="{{widget.name}}" id="union_{{choice.widget.name}}" value="{{choice.name}}"{% if widget.required %} required{% endif %}{% if choice.checked %} checked{% endif %}{% if widget.disabled %} disabled{% endif%}>
+    <label for="union_{{choice.widget.name}}">{# TODO: help text as tooltip ??? #}{{choice.label}}</label>
+    <div class="union-field-content">{% include choice.widget.template_name with widget=choice.widget %}</div>
+  </div>
 {% endfor %}
 {% endwith %}
-</ul>
\ No newline at end of file
+</div>
```

Comment:
1. ul/li often have predefined css that depends of the browser/os/moon size/... so I usually prefer div. In this case the css rule is really strict so why not.
2. In css it is more common to use sub-widget or subwidget instead of sub_widget, and the widget term is everywhere so it is confusing too. I'm ok with `union-cases + union-case`